lib/oembed/provider.rb

```diff
@@ -135,7 +135,7 @@ def raw(url, query = {})
         http.verify_mode = OpenSSL::SSL::VERIFY_PEER
         http.read_timeout = http.open_timeout = query[:timeout] if query[:timeout]
         
-        %w{scheme userinfo host port registry}.each { |method| uri.send("#{method}=", nil) }
+        %w{scheme userinfo host port}.each { |method| uri.send("#{method}=", nil) }
```

Comment:
Since I have so little time to maintain this, I always hedge at introducing backwards incompatibility. Any chance you could update this code so that it only ignores the registry option in Ruby 2.2?